documentation/asciidoc/topics/ref_upgrades.adoc

```diff
@@ -26,3 +26,25 @@ ifdef::downstream[]
 If you upgrade {brandname} clusters manually and have upgraded the channel for your {ispn_operator} subscription from **8.1.x** to **8.2.x** you should apply the upgrade for the latest {brandname} 8.2.x version as soon as possible to avoid potential data loss that can result from an issue in 8.2.0.
 For more information, see link:{relnotes_openshift}[{ispn_operator} {ProductVersion} Release Notes].
 endif::downstream[]
+
+[discrete]
+== Upgrade types
+//Community Only
+ifdef::community[]
+The Infinispan CR path ```spec.upgrades.type``` defines how the upgrades are done:
+
+[%header,cols=2*]
+|===
+|Value
+|Description
+
+|`HotRodRolling`
+|Upgrades the cluster with no downtime using Hot Rod Rolling Upgrades. The operator first creates a new statefulSet with pods in the new version, that use existing pods as a remote store; then redirects all clients to the new pods while copying all the data from the old cluster, disposing the original pods in the end.
```

Comment:
"Upgrades the cluster with no downtime using Hot Rod Rolling Upgrades. The operator first creates a new statefulSet with pods in the new version, that use existing pods as a remote store; then redirects all clients to the new pods while copying all the data from the old cluster, disposing the original pods in the end."

Upgrades {brandname} clusters without downtime. {ispn_operator} creates a new {brandname} cluster with the target version and uses a remote store to copy data from the existing cluster. During the upgrade {ispn_operator} also redirects all clients to the new cluster. When it completes the upgrade process, {ispn_operator} removes the old {brandname} cluster.